### PR TITLE
mm-19, mm-20, mm-21

### PR DESCRIPTION
**Pull request by nandinip92**
mm-19: Fetch Data from API
mm-20: Displaying the image and explanation on the screen.
mm-21: Styling the APOD for desktop
APODDesktop

mm-25: Implementing ~Like and Dislike button
Like-Dislike

### CODE REVIEW

File: src/components/APOD/styles.scss
Comment (Louisasa) [inline src/components/APOD/styles.scss]: remove this comment please

File: src/components/APOD/styles.scss
Comment (Louisasa) [inline src/components/APOD/styles.scss]: and remove this border please :)

**Review by Louisasa — COMMENTED** [review]
looking good, have you checked how this looks in mobile/tablet?

File: src/components/APOD/styles.scss
Comment (nandinip92) [reply src/components/APOD/styles.scss]: done!

File: src/components/APOD/styles.scss
Comment (sailormoonvicky) [reply src/components/APOD/styles.scss]: done!

File: src/components/APOD/apod.test.tsx
Comment (Louisasa) [inline src/components/APOD/apod.test.tsx]: remove this please

File: src/components/APOD/apod.test.tsx
Comment (Louisasa) [inline src/components/APOD/apod.test.tsx]: and this please

File: src/components/APOD/loading_spinner.tsx
Comment (Louisasa) [inline src/components/APOD/loading_spinner.tsx:6]: can we center the loading spinner? `display: flex;` and `justify-content: center;` should do it

File: src/components/APOD/styles.scss
Comment (Louisasa) [inline src/components/APOD/styles.scss]: can we remove this please

File: src/components/Layouts/MarsRoverPage/index.tsx
Comment (Louisasa) [inline src/components/Layouts/MarsRoverPage/index.tsx:13]: sorry this was me, would you mind removing while you do the other markups please! thank you!

**Review by Louisasa — COMMENTED** [review]
I fixed the hover issues, it was the z-index on the landing page which we don't need anymore :) only some small styling issues now and then it looks good to merge :)

File: src/components/APOD/apod.test.tsx
Comment (sailormoonvicky) [reply src/components/APOD/apod.test.tsx]: Done!

File: src/components/APOD/apod.test.tsx
Comment (sailormoonvicky) [reply src/components/APOD/apod.test.tsx]: done!

File: src/components/APOD/loading_spinner.tsx
Comment (sailormoonvicky) [reply src/components/APOD/loading_spinner.tsx:6]: Done!

File: src/components/APOD/styles.scss
Comment (sailormoonvicky) [reply src/components/APOD/styles.scss]: Done!

File: src/components/Layouts/MarsRoverPage/index.tsx
Comment (sailormoonvicky) [reply src/components/Layouts/MarsRoverPage/index.tsx:13]: Done!

File: src/components/APOD/apod.test.tsx
Comment (Louisasa) [reply src/components/APOD/apod.test.tsx]: it's still here?

File: src/components/APOD/like_dislike_button.tsx
Comment (Louisasa) [inline src/components/APOD/like_dislike_button.tsx:1]: hey so you've added these changes to the same branch and instead of making a new one, so we're gonna have to wait for these changes until we can merge it all in

File: src/components/APOD/like_dislike_button.tsx
Comment (Louisasa) [inline src/components/APOD/like_dislike_button.tsx]: can we remove these comments please?

File: src/components/APOD/styles.scss
Comment (Louisasa) [inline src/components/APOD/styles.scss:105]: can we use the tablet/desktop already defined

Comment (Louisasa) [reply src/components/APOD/styles.scss:105]: still this here please

File: src/components/APOD/picture_tile.tsx
Comment (Louisasa) [inline src/components/APOD/picture_tile.tsx]: can we add styling on this and the overlay info text too?

**Review by Louisasa — APPROVED** [review]

**Pull request merged**
